user/docker.md

```diff
@@ -22,9 +22,11 @@ services:
 Then you can add `- docker` commands to your build as shown in the following
 examples.
 
-> Travis CI automatically routes builds to run on Trusty `sudo: required` when `services: docker` is configured.
+> Travis CI automatically routes builds to run on our Trusty sudo-enabled infrastructure when `services: docker` is configured.
 > We do not currently support use of Docker on OS X.
 
+> For information on how to use Docker on Travis CI Enterprise check out [https://docs.travis-ci.com/user/enterprise/build-images/#Enabling-Docker-Builds](https://docs.travis-ci.com/user/enterprise/build-images/#Enabling-Docker-Builds)
```

Comment:
Especially in the note, just using the URL is quite hard on the eye, I'd suggest

> For information on how to use Docker on Travis CI Enterprise check out [https://docs.travis-ci.com/user/enterprise/build-images/#Enabling-Docker-Builds](Enabling Docker Builds)